apps/src/lib/tools/jsdebugger/JsDebugger.jsx

```diff
@@ -120,10 +120,19 @@ export const UnconnectedJsDebugger = Radium(React.createClass({
       watchersHidden: false,
       open: this.props.isOpen,
       openedHeight: 120,
+      labelDisplay: 'inline'
     };
   },
 
+  handleWindowResize() {
+    let hideLabel = ($(window).width() <= 1275) && !this.state.watchersHidden;
+    let labelDisplay= hideLabel ? 'none' : 'inline';
```

Comment:
Alternatively (and possibly even better) you could have the state be windowWidth, and then have the style be `{[this.state.windowWidth <= 1275 && styles.hidden}]`